.pylintrc

```diff
@@ -2,6 +2,8 @@
 # Add files or directories to the ignore list. They should be base names, not
 # paths.
 ignore=CVS
+generated-members=torch.*
```

Comment:
I'm pretty sure this isn't the best way to do this, but without it, pylint will complain about `torch` not having the right members everywhere. I believe this is because `torch` _could_ be None, though it never actually happens.